linera-core/src/client.rs

```diff
@@ -739,6 +787,7 @@ where
         Ok(info)
     }
 
+    #[tracing::instrument(level = "trace", skip(committee, certificate))]
```

Comment:
Optional: maybe adding the certificate hash here could be useful.

```suggestion
    #[tracing::instrument(
        level = "trace",
        skip(committee, certificate),
        fields(certificate_hash = certificate.hash())
    )]
```

Reply:
I'm preferring to err on the side of fewer fields than more right now, I think.  If they turn out to be useful to debug a particular problem, it's relatively easy to add them later on.